docs/features.rst

```diff
@@ -17,12 +17,44 @@ desktops. It boasts a well documented library interface that is easy to
 use. It comes with a simple bittorrent client demonstrating the use of
 the library.
 
+BitTorrent v2 is supported as of libtorrent 2.0. This replaces the previous
+merkle hash tree extension.
+
 features
 ========
 
 libtorrent is an ongoing project under active development. Its
 current state supports and includes the following features:
 
+BitTorrent v2
+-------------
+
+Starting with version 2.0, libtorrent supports BitTorrent V2 (as specified in
+`BEP 52`_). BitTorrent V2 introduces a new format for .torrent files, whish is
+smaller than the original format.
+
+1. Files are organized in a directory structure, instead of listing full paths.
+   Torrents that have a lot of files in deep directory structures will use a lot
+   less space to represent that structure in a v2 torrent.
+
+2. Piece hashes are organized in a merkle hash trees per file, and only the
+   roots of the trees are included in the .torrent file. The actual hashes are
+   delivered by peers.
```

Comment:
Not just the roots are in the .torrent file, the piece layer hashes are stored as well, albeit outside of the info-dict. This also impacts the claim above about v2 torrent files being smaller. In the common case where the torrent only contains a handful of files the v2 torrent will be larger due to the larger hashes.

Reply:
Good point